### PR TITLE
feature/add-view

### PR DESCRIPTION
**Pull request by khorzhenwin**
Base UI has been created with scraping and validation. Idk where its supposed to add the doujin id to cause i've been told "db be gone"

### CODE REVIEW

**Review by wooneusean — CHANGES_REQUESTED** [review]
Please add your `.vscode` file to the `.gitignore` thanks

**Review by wooneusean — APPROVED** [review]
lgtm

File: lib/views/add_dialog.dart [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      ),
+      actions: [
+        TextButton(
+          child: Text('SUBMIT'),
+          onPressed: nukeCodes.text.isNotEmpty
+              ? () async {
+                  try {
+                    var doujin = await DoujinHelper.scrape(
+                        int.parse(nukeCodes.text.trim()));
+                    // var doujinProvider = DoujinProvider();
```
Comment (wooneusean) [inline lib/views/add_dialog.dart:94]: Please uncomment this two lines, it is a core feature of your task.

**Review by wooneusean — CHANGES_REQUESTED** [review]
Please address the above comments

File: lib/views/add_dialog.dart [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+class AddDialog extends StatefulWidget {
+  const AddDialog({Key? key}) : super(key: key);
+
+  @override
+  State<AddDialog> createState() => _AddDialogState();
+}
+
+class _AddDialogState extends State<AddDialog> {
+  final nukeCodes = TextEditingController();
```
[the remark this answers is not in the document]
Comment (khorzhenwin) [reply lib/views/add_dialog.dart]: renamed nukeCodes -> _nukeCode

File: lib/views/add_dialog.dart (hunk 1 above)
Comment (khorzhenwin) [reply lib/views/add_dialog.dart]: The 2 lines have been uncommented and tested with retrieval from localdb + modified error message in failed snack bar to show exception message.

**Review by wooneusean — APPROVED** [review]
lgtm

**Pull request merged**